node/src/neighborhood/mod.rs

```diff
@@ -3898,99 +4004,74 @@ mod tests {
-                    neighborhood
-                        .neighborhood_database
-                        .node_by_key(&at_public_key)
-                        .unwrap()
-                        .metadata
-                        .country_undesirability,
+                    neighborhood.get_node_country_undesirability(&at_public_key),
                     UNREACHABLE_COUNTRY_PENALTY,
-                    "at We expecting {}, country is considered for exit location in fallback",
+                    "at We expect {}, country is considered for exit location in fallback",
                     UNREACHABLE_COUNTRY_PENALTY
                 );
                 assert_eq!(
-                    neighborhood
-                        .neighborhood_database
-                        .node_by_key(&pl_public_key)
-                        .unwrap()
-                        .metadata
-                        .country_undesirability,
+                    neighborhood.get_node_country_undesirability(&pl_public_key),
                     UNREACHABLE_COUNTRY_PENALTY,
-                    "pl We expecting {}, country is too close to be exit",
+                    "pl We expect {}, country is too close to be exit",
                     UNREACHABLE_COUNTRY_PENALTY
                 );
             }),
         };
         subject_addr.try_send(BindMessage { peer_actors }).unwrap();
 
         subject_addr.try_send(message).unwrap();
-        subject_addr.try_send(assertion_msg).unwrap();
 
+        subject_addr.try_send(assertion_msg).unwrap();
         System::current().stop();
         system.run();
-
-        //println!("recorder: {:#?}", &recorder.try_into().unwrap());
         let exit_location_recording = &arc_recorder.lock().unwrap();
-        let exit_handler_response = exit_location_recording
-            .get_record::<NodeToUiMessage>(0)
-            .body
-            .payload
-            .clone();
         let log_handler = TestLogHandler::new();
         assert_eq!(
-            exit_handler_response,
-            Err((9223372036854775817, "CZ, SK, IN".to_string(),))
+            exit_location_recording
+                .get_record::<NodeToUiMessage>(0)
+                .body,
+            MessageBody {
+                opcode: "exitLocation".to_string(),
+                path: MessagePath::Conversation(123),
+                payload: Err((9223372036854775816, "CZ, SK, IN".to_string(),))
+            }
+        );
+        assert_eq!(
+            exit_location_recording
+                .get_record::<NodeToUiMessage>(0)
+                .target,
+            MessageTarget::ClientId(234)
         );
         log_handler.assert_logs_contain_in_order(vec![
             &format!(
                 "INFO: {}: Fallback Routing is set. Exit location set:",
                 test_name
             ),
-            &"Country Codes: [\"CZ\", \"SK\", \"IN\"] - Priority: 1;",
+            &"Country Codes: [\"CZ\", \"SK\", \"IN\"] - Priority: 1",
```

Comment:
Tohle je problematicke z toho duvodu, ze ten log nezacina tou svoji hlavickou, kde by se objevil prave i nazev testu jakozto jmeno prirazene tomu loggeru. Takhle, ciste hypoteticky, muze tenhle log interferovat s logem z jineho testu. (Proto se to dela, ze davame loggerum nazvy podle toho testu).